heclib/javaHeclib/src/Hec_ztsStore.c

```diff
@@ -234,8 +234,8 @@ JNIEXPORT jint JNICALL Java_hec_heclib_util_Heclib_Hec_1ztsStore(
 
 
 	//  timeGranularitySeconds
-	//  Number of seconds each unit in times array has, normally 60 (for one minute)
-	//  Can be 1, 60, 3600, 86400
+	//  Number of seconds each unit in times array has, normally SECS_IN_1_MINUTE (for one minute)
```

Comment:
please consider using:
#define SECOND_GRANULARITY 1
#define MINUTE_GRANULARITY 60
#define HOUR_GRANULARITY 3600
#define DAY_GRANULARITY 86400

Reply:
Sure.

Reply:
They were already defined and used, except for the SECS_IN_1_MINUTE mistakes above. And comments.